pkg/kv/kvpb/api.go

```diff
@@ -2100,6 +2115,11 @@ func (r *LinkExternalSSTableRequest) flags() flag {
 	}
 	return flags
 }
+
+func (r *ExciseRequest) flags() flag {
+	return isWrite | isRange | isAlone | bypassesReplicaCircuitBreaker
```

Comment:
Should it bypass the circuit breaker? It probably doesn't hurt, but we don't expect it to have to either, correct? Ranges should not become unavailable as a result of missing user keys.